src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+}
+
+.stars__star--first {
+  margin-left: 0;
+}
+
+.stars__star--active {
```

Comment:
use stars modifier and nth-child selector
for example `.stars--2 :nth-child(-n+2)`